### ISSUE TITLE
Recording voice messages not working

### ISSUE THREAD

**Issue by agasoft** [UnigramDev/Unigram#2285]
Hello,
in latest Unigram versions, I cannot record voice messages. I am using screen reader to operate since I am blind. Shortcut key ctrl+r not working, but not working voice message button as well.
Microphone is allowed.
Tried to reinstall but its still not working.

**Comment by sukiletxe** [comment]
@agasoft reported by another person too. It works for me

**Comment by MuhammadGagah** [comment]
Try to change your default input, sometimes Unigram doesn't support the default input you use. To change input, press start menu, and type mmsys.cpl, and press enter. after sound windo open, press control+tab one time, and set to default input what you want.

**Comment by agasoft** [comment]
No, its not the case.
Tried and its not working.

2021-03-20 1:04 GMT+01:00, MuhammadGagah ***@***.***>:

**Comment by gajender-del** [comment]
@agasoft After reinstallation of the program issue is not fixed

**Comment by MarcAnt01** [comment]
Well, reinstalling an app doesn't help to fix this kind of issues usually 😅

**Comment by gajender-del** [comment]
@MarcAnt01 so, what can we do to resolve the issue?

**Comment by MarcAnt01** [comment]
I have no clue at all, recording works properly in my devices

**Comment by FrayxRulez** [comment]
@MarcAnt01 It doesn’t work for me neither. Might this depend on OS version?

**Comment by MarcAnt01** [comment]
I have one laptop with 20H1 and the other with 20H2 and it works on both

**Comment by gajender-del** [comment]
@MarcAnt01 I cannot record voice messages after march 14th.

**Comment by MarcAnt01** [comment]
I guess it's more useful to know the version number that brought the issue instead

**Comment by gajender-del** [comment]
@MarcAnt01 I am using Unigram7.5.5971.0 and iam using Windows ten v20 h2.

**Comment by sukiletxe** [comment]
OK, now it failed for me. No version update nor windows update in between (this morning it worked just fine, tested with saved messages, now it fails).

**Comment by larsschellhas** [comment]
Yeah, I'm having the same issues with 
- Unigram 7.5 (5972) X64
- Windows 10 20H2 19042.844
- Windows Feature Experience Pack 120.2212.551.0

I can see that the microphone button in the task bar pops up when I first click the voice message button. 
Also: Were video messages removed? Because I can also not switch between video and voice messages with single click.
Like you all, I noticed the behaviour a few days ago, but I'm not sure whether it came with a system update or unigram update.

**Comment by FrayxRulez** [comment]
Video messages were removed in 2018.
I have the same issue as well. Might it depend on the experience pack? What is see is that the file gets locked by something right after it gets created, so the app can't actually access it.

**Comment by MarcAnt01** [comment]
> Video messages were removed in 2018.
> I have the same issue as well. Might it depend on the experience pack? What is see is that the file gets locked by something right after it gets created, so the app can't actually access it.

Same experience pack as @larsschellhas and working in all my devices

**Comment by gajender-del** [comment]
@MarcAnt01 in this case what we have to do

**Comment by MarcAnt01** [comment]
> @MarcAnt01 in this case what we have to do

Wait and hope that Fela will find out something more in order to fix it

**Comment by FrayxRulez** [comment]
Fixed in https://github.com/UnigramDev/Unigram/commit/b845d6d36d3cda1bbc97535b588e427a00185a0a, I hope.

**Comment by larsschellhas** [comment]
Just confirming. It's solved for me :D
